pkg/controllers/flannelmigration/migration_controller.go

```diff
@@ -370,14 +370,17 @@ func (c *flannelMigrationController) runIpamMigrationForNodes() ([]*v1.Node, err
 	for _, obj := range items {
 		node := obj.(*v1.Node)
 
-		migrationInProcess, _ := getNodeLabelValue(node, migrationNodeInProcessKey)
+		migrationInProgress, _ := getNodeLabelValue(node, migrationNodeInProgressKey)
 		network, _ := getNodeLabelValue(node, migrationNodeSelectorKey)
 
-		if network != "calico" || migrationInProcess == "true" {
-			n := k8snode(node.Name)
-			if err := n.addNodeLabels(c.k8sClientset, nodeNetworkFlannel); err != nil {
-				log.WithError(err).Errorf("Error adding node label to node %s.", node.Name)
-				return []*v1.Node{}, err
+		if network != "calico" || migrationInProgress == "true" {
+			if network != "calico" {
```

Comment:
Should we skip if `network==none` as well?

Reply:
Code updated.